### PR TITLE
add new resource iam_folders_policy_binding. addresses hashicorp/terraform-provider-google/issues/20198

### PR DESCRIPTION
**Pull request by derekchu-google**
```release-note:new-resource
`google_iam_folders_policy_binding` (beta)
```

### CODE REVIEW

**Comment by derekchu-google** [comment]
PR is currently WIP

**Comment by hao-nan-li** [comment]
> PR is currently WIP

Could you change it as a draft, and re-open it when it's ready?

File: mmv1/products/iam3/FoldersPolicyBinding.yaml [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+self_link: 'folders/{{folder}}/locations/{{location}}/policyBindings/{{policy_binding_id}}'
+create_url: 'folders/{{folder}}/locations/{{location}}/policyBindings?policyBindingId={{policy_binding_id}}'
+update_verb: 'PATCH'
+update_mask: true
+import_format:
+  - 'folders/{{folder}}/locations/{{location}}/policyBindings/{{policy_binding_id}}'
+timeouts:
+  insert_minutes: 20
+  update_minutes: 20
+  delete_minutes: 20
```
Comment (hao-nan-li) [inline mmv1/products/iam3/FoldersPolicyBinding.yaml:33]: Just a quick comment on this, do you happen to know how long does it to create this resource?

Comment (derekchu-google) [reply mmv1/products/iam3/FoldersPolicyBinding.yaml:33]: it should only take a few seconds but with regional propagation, i think we say our SLOs are 7 minutes.

Comment (hao-nan-li) [reply mmv1/products/iam3/FoldersPolicyBinding.yaml:33]: Sounds good.

**Review by hao-nan-li — APPROVED** [review]

**Pull request merged**